tests/conftest.py

```diff
@@ -5293,3 +5294,44 @@ def set_live_must_gather_images(pytestconfig):
         config.REPORTING[
             "default_ocs_must_gather_latest_tag"
         ] = defaults.MUST_GATHER_UPSTREAM_TAG
+
+
+@pytest.fixture(scope="function")
+def create_resources_using_kube_job(request):
+    """
+    Create resources using k8s fixture. This fixture makes use of the FioPodScale class
+    to create the expected number of POD+PVC
+    """
+
+    instances = []
+
+    def factory(
+        scale_pvc=200,
+        pvc_per_pod_count=20,
+        start_io=True,
+        io_runtime=None,
+        pvc_size=None,
+        max_pvc_size=50,
+    ):
+        # Scale FIO pods in the cluster
+        fioscale = FioPodScale(
+            kind=constants.DEPLOYMENTCONFIG, node_selector=constants.SCALE_NODE_SELECTOR
+        )
+        instances.append(fioscale)
+        kube_pod_obj_list, kube_pvc_obj_list = fioscale.create_scale_pods(
+            scale_count=scale_pvc,
+            pvc_per_pod_count=pvc_per_pod_count,
+            start_io=start_io,
+            io_runtime=io_runtime,
+            pvc_size=pvc_size,
+            max_pvc_size=max_pvc_size,
+        )
+        return kube_pod_obj_list, kube_pvc_obj_list
+
+    def finalizer():
+        log.info("Cleaning the fioscale instances")
+        for instance in instances:
+            instance.cleanup()
```

Comment:
Does all the resources are getting cleaned up here?. Hope we are not leaving any left overs.